setup.py

```diff
@@ -20,6 +20,7 @@
     author_email='dev@praekeltfoundation.org',
     packages=find_packages(),
     install_requires=[
+        'six==1.10.0',
```

Comment:
Is there any particular reason to depend on a specific version? In general, requirements for libraries should be as broad as possible to avoid potential conflicts.

Reply:
Only that I could remember the syntax for pinning to a specific version without looking it up :). Amended!

Reply:
Oh and thanks for taking a look at this pull request!